src/SourceBuild/patches/emsdk/0001-main-Update-dependencies-from-dotnet-arcade-1143.patch

```diff
@@ -0,0 +1,26 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "dotnet-maestro[bot]"
+ <42748379+dotnet-maestro[bot]@users.noreply.github.com>
+Date: Mon, 24 Mar 2025 11:40:11 +0000
+Subject: [PATCH] [main] Update dependencies from dotnet/arcade (#1143)
+
+[main] Update dependencies from dotnet/arcade
+
+
+ - Remove duplicate .msi FileExtensionSignInfo
+---
```

Comment:
Per the [patching guidelines](https://github.com/dotnet/source-build/blob/main/Documentation/patching-guidelines.md), please include a `BackPort: ` link to an issue/PR that tracks the removal of this patch.  This aids in tracking patch removal.